microservices/device-simulation-dotnet/deployment/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "@azure-iot/tsconfig",
```

Comment:
you are taking a dependency on azure iot.
I dont think sim should take this dependency.
you should use some generally available recommended tsconfig package

Reply:
Removed. Those files shouldn't have even been in this branch. Good catch.